csc302-spring-2021/proj-DaTeam
# fix: CI workflows

## Description



- Docker builds for backend are currently broken because the @dateam/shared module is not present in the container
- This PR modifies Dockerfiles to copy shared module into the container
- Also include changes github actions workflows to use new script commands

## Checklist


## QA Steps



1. `npm run clean:ci`
1. `npm install`
2. `npm run docker:build`
3. `npm run docker:up`
3. Verify that http://localhost:3000 shows frontend and http://localhost:3001 shows backend
